tutorials/animation/introduction_2d.rst

```diff
@@ -16,11 +16,10 @@ In this guide you learn to:
 -  Create a simple animation
 -  Call functions with the powerful Call Function Tracks
 
-In Godot you can animate anything you find in the Inspector. Animations
-are changes to properties over time. This means you can animate anything
-visual in nature: sprites, UI elements, particles, the visibility, and
-color of textures. But not only. You can also control gameplay related
-values and call any function.
+In Godot, you can animate anything available in the Inspector, such as
+Node transforms, sprites, UI elements, particles, visibility and colour 
```

Comment:
We use en_US as reference spelling for the documentation, so this should be spelled "color". I'll push a fixup and merge.